Readme.md

```diff
@@ -4,6 +4,17 @@
 
 This GitHub repository is meant for collecting `pytest` example code to introduce colleagues to `pytest`.
 
+Pytest is a python based testing framework, whichis used to write and execute test code. You can write code to test anything like database , API. But in the industry pytest is mainly used to write tests for APIs. 
+
+# Advantages
```

Comment:
maybe you can add that pytest works in combination with existing test suite like nosetest, python unittest, doctest, ... If you already have tests you can easily switch to pytest.